code/common/merge.q

```diff
@@ -72,7 +72,7 @@ mergebypart:{[tablename;dest;partchunks]
    .lg.o[`merge;"upserting ",(string count chunks)," rows to ",string dest];
    /-merge columns to permanent storage
    .[upsert;(dest;chunks);                     
-     {.lg.e[`merge;"failed to merge to ", string[dest], " from segments ", (", " sv string chunks)];}];
+     {[e;d;p].lg.e[`merge;raze "failed to merge to ", string[d], " from segments ", (", " sv string p), " Error is - ",string[]e]}[;dest;partchunks]];
```

Comment:
do we need the `raze`? if we're constructing a single string should be needed I don't think?

Reply:
hmm fair point - I _think_ this was due to the partchunks - let me review here

Reply:
string[]e

^ epic fail on this as well